### PR TITLE
moves ccf reference into new persistent reference bucket

### PR DESCRIPTION
**Pull request by MatthewAitken**

### CODE REVIEW

File: pipeline/deploy/cloudformation/morphology_pipeline.yml [hunk 1]
```diff
@@ -775,6 +775,9 @@ Resources:
                             "Name": "WORKING_BUCKET",
                             "Value": "${WorkingBucket}" 
                           },
+                          {
+                            "Name": "REFERENCE_BUCKET",
+                            "Value": "${PersistentStackName}-reference"
```
Comment (NileGraddis) [inline pipeline/deploy/cloudformation/morphology_pipeline.yml:780]: missing a `}`?

**Review by NileGraddis — APPROVED** [review]
1 comment, otherwise lgtm

**Comment by NileGraddis** [comment]
another thing that occurs to me: the setup instructions should now include a snippet about adding reference information.

File: pipeline/deploy/cloudformation/morphology_pipeline.yml (hunk 1 above)
Comment (MatthewAitken) [reply pipeline/deploy/cloudformation/morphology_pipeline.yml:780]: good catch, it wasn't building but not giving me a useful error.

**Review by sgratiy — APPROVED** [review]

**Review by JakeSAI — COMMENTED** [review]
LGTM!

**Pull request merged**